submissions/kolajoTomike/htmlForms/contact.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Page</title>
+</head>
+<body>
+<h1>The Berry Ice Cream</h1>
+
+           <h2>Contact Us</h2>
+           <input type="text" name="Full Name" id="Full Name" placeholder="Full Name"><br><br><br>
```

Comment:
@tomike19 update the name of the name, id of the input field to be in lowercase and no space between each word.